tendermint/src/abci/response/check_tx.rs

```diff
@@ -31,21 +31,115 @@ pub struct CheckTx {
     pub events: Vec<Event>,
     /// The namespace for the `code`.
     pub codespace: String,
-    /// The transaction's sender (e.g. the signer).
-    pub sender: String,
-    /// The transaction's priority (for mempool ordering).
-    pub priority: i64,
-    /// mempool_error is set by Tendermint.
-    /// ABCI applications should not set mempool_error.
-    pub mempool_error: String,
+    // FIXME: Should we still support these fields for older protocol versions?
+    // Does anybody use them?
+    //pub sender: String,
+    //pub priority: i64,
+    //pub mempool_error: String,
```

Comment:
They're still used in v0.34 (still running on most mainnets), and v0.37 (still gaining traction/adoption). They relate to the priority mempool, which was marked as deprecated in v0.37 and removed in v0.38.

So I think they should still be present in the domain type while we support v0.34 and v0.37.

Reply:
https://github.com/informalsystems/tendermint-rs/pull/1312/commits/1f5688ddd542c7bfcd46b34b10c22eec0f7f2e0f